app/javascript/components/ActionItemSelectParticipants/index.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import { Button, Dialog, DialogContent } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
+import ActionItemSearchParticipants from '../ActionItemSearchParticipants';
+import ActionItemDisplayParticipants from '../ActionItemDisplayParticipants';
+import styles from './styles';
+
+class ActionItemSelectParticipants extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      participants: this.props.participants,
+      selectedParticipants: [],
+      openModal: false,
+    };
+    this.addUserToState = this.addUserToState.bind(this);
+    this.removeUserFromState = this.removeUserFromState.bind(this);
+    this.addAllUsersToState = this.addAllUsersToState.bind(this);
+    this.removeAllUsersFromState = this.removeAllUsersFromState.bind(this);
+  }
+
+  // Adds selected user to state to be displayed
+  addUserToState(user) {
+    this.setState(prevState => ({
+      selectedParticipants: [...prevState.selectedParticipants, user],
+    }));
+  }
+
+  // Removes user from display
+  removeUserFromState(user) {
+    this.setState(prevState => {
+      const copy = [...prevState.selectedParticipants];
+      const index = this.state.selectedParticipants.indexOf(user);
+      copy.splice(index, 1); // Removes one element at `index` location
+      return { selectedParticipants: copy };
+    });
+  }
+
+  // Adds all users at once to be displayed
+  addAllUsersToState() {
+    this.setState(prevState => ({
+      selectedParticipants: prevState.participants,
+    }));
+  }
+
+  // Removes all users at once from display
+  removeAllUsersFromState() {
+    this.setState({
+      selectedParticipants: [],
+    });
+  }
+
+  toggleModal() {
+    this.setState(prevState => ({
+      openModal: !prevState.openModal,
+    }));
+  }
+
+  render() {
+    const { classes } = this.props;
+    return (
+      // Overall component
+      <div className={classes.entirePage}>
+        <Button
+          variant="outlined"
+          color="primary"
+          onClick={() => this.toggleModal()}
+        >
+          Open Modal
+        </Button>
+        <Dialog
+          onClose={() => this.toggleModal()}
+          open // {this.state.openModal} // true
+          fullWidth
+          maxWidth="lg"
+          style={
+            {
+              // minHeight: '60vh',
+              // maxHeight: '60vh',
+            }
+          }
+        >
+          <DialogContent style={{ overflow: 'hidden' }}>
+            Create New Assignment List
+            <div
```

Comment:
The styling for these two components is off on my screen: the search component overlaps the display component. This is probably related to having different screen sizes so you should check out the styling when your screen is zoomed in and out.